bun/starter/src/main.ts

```diff
@@ -1,29 +1,31 @@
-import { Client } from "node-appwrite";
+import { Client, Users } from "node-appwrite";
 
-// This is your Appwrite function
-// It's executed each time we get a request
+// This Appwrite function will be executed every time your function is triggered
 export default async ({ req, res, log, error }: any) => {
-  // Why not try the Appwrite SDK?
-  //
-  // const client = new Client()
-  //    .setEndpoint('https://cloud.appwrite.io/v1')
-  //    .setProject(Bun.env["APPWRITE_FUNCTION_PROJECT_ID"])
-  //    .setKey(Bun.env["APPWRITE_API_KEY"]);
+  // You can use the Appwrite SDK to interact with other services
+  // For this example, we're using the Auth service
+  const client = new Client()
+    .setEndpoint(Bun.env["APPWRITE_FUNCTION_API_ENDPOINT"])
+    .setProject(Bun.env["APPWRITE_FUNCTION_PROJECT_ID"])
+    .setKey(req.headers['x-appwrite-key'] ?? '');
+  const users = new Users(client);
 
-  // You can log messages to the console
-  log("Hello, Logs!");
-
-  // If something goes wrong, log an error
-  error("Hello, Errors!");
+  try {
+    const response = await users.list();
+    // Log messages and errors to the Appwrite Console
+    // These logs won't be seen by your end users
+    log(`Amount of users: ${response.total}`);
+  } catch(err) {
+    error("Could not list users: " + err.message);
+  }
 
   // The `req` object contains the request data
```

Comment:
```suggestion
  // The req object contains the request data
```
This isnt markdown so this doesn't really add any meaning here